### PR TITLE
Added setMouseNavEnabled() support in Navigator

### PR DESCRIPTION
**Pull request by msalsbery**
Fix for #572

### CODE REVIEW

**Comment by iangilman** [comment]
Looks reasonable, though it makes me wonder why the navigator has this.element.innerTracker rather than replacing this.innerTracker. What do you think?

**Comment by msalsbery** [comment]
I agree. I saw that and refreshed my memory on how the existing trackers are disabled and couldn't find a good reason for doing it that way. I'll fix it.=

**Comment by msalsbery** [comment]
@iangilman  Fixed. Now replaces base class' innerTracker with a Navigator-specific one

File: src/navigator.js [hunk 1]
```diff
@@ -223,8 +200,26 @@ $.Navigator = function( options ){
...
+    this.innerTracker.destroy();
+    this.innerTracker = new $.MouseTracker({
+        element:         this.element,
+        dragHandler:     $.delegate( this, onCanvasDrag ),
+        clickHandler:    $.delegate( this, onCanvasClick ),
+        releaseHandler:  $.delegate( this, onCanvasRelease ),
+        scrollHandler:   $.delegate( this, onCanvasScroll )
+    }).setTracking( true );
+
+    /*this.displayRegion.outerTracker = new $.MouseTracker({
```
Comment (iangilman) [inline src/navigator.js]: Maybe we should kill this commented-out code?

File: src/navigator.js [hunk 2]
```diff
@@ -211,10 +192,6 @@ $.Navigator = function( options ){
 
     this.displayRegionContainer.appendChild(this.displayRegion);
     this.element.getElementsByTagName('div')[0].appendChild(this.displayRegionContainer);
-    unneededElement = this.element.getElementsByTagName('textarea')[0];
```
Comment (iangilman) [inline src/navigator.js]: Wow.

**Comment by iangilman** [comment]
I assume you've verified that the navigator still works with this change? 

If you merge in the latest from master, it'll fix the changelog conflict. Otherwise I'll do it when I merge.

File: src/navigator.js (hunk 2 above)
Comment (msalsbery) [inline src/navigator.js]: Beats me :)

File: src/navigator.js (hunk 1 above)
Comment (msalsbery) [inline src/navigator.js]: Okey dokey. It's been in there so long I thought there was a reason :)

**Comment by msalsbery** [comment]
Yes, but I'll do the merge, update the changelog, and test again to make sure.

**Pull request merged**